cartoframes/data/observatory/enrichment/enrichment_service.py

```diff
@@ -378,11 +378,18 @@ def _is_subscribed(dataset, geography, credentials):
 
 
 def _get_aggregation(variable, aggregation):
-    if aggregation == AGGREGATION_NONE:
-        return None
+    if aggregation in [None, AGGREGATION_NONE]:
+        aggregation_method = None
     elif aggregation == AGGREGATION_DEFAULT:
-        return variable.agg_method
+        aggregation_method = variable.agg_method
     elif isinstance(aggregation, str):
-        return aggregation
+        aggregation_method = aggregation
     elif isinstance(aggregation, dict):
-        return aggregation.get(variable.id, variable.agg_method)
+        aggregation_method = aggregation.get(variable.id, variable.agg_method)
+    else:
+        raise ValueError('The `aggregation` parameter is invalid.')
+
+    if aggregation_method is None:
```

Comment:
We can refactor this:

```py
if aggregation_method is not None:
    return aggregation_method.lower()
```

But it's a minor one

Reply:
:+1: